config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-0-upgrades.yaml

```
@@ -0,0 +1,221 @@
+periodics:
+
+- name: periodic-cluster-api-e2e-workload-upgrade-1-18-1-19-release-1-0
+  interval: 48h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: release-1.0
+    path_alias: sigs.k8s.io/cluster-api
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211022-00550799a0-1.21
```

```suggestion
    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211022-00550799a0-1.22
```
Please upgrade to 1.22 in all release-1.0 files (the idea is that we use the same version which we use in CAPI, CAPI v1.0.0 is using controller-runtime 0.10 which is using Kubernetes 1.22)